### ISSUE TITLE
Editing small icons, images, ect

### ISSUE THREAD

**Issue by oleteacher** [givanz/VvvebJs#75]
When working with very small images (like svg), icons or fontawesome icons hard to get settings. Example:

![image](https://user-images.githubusercontent.com/26278259/58201020-b9c44900-7c99-11e9-901e-b2750c4b3df8.png)

Seems the plus icon is placed over them and every time end up adding new element instead of being able to edit:)

**Comment by aloksharma1** [comment]
yes if the plus icon can be pushed to bottom of blue border this issue will be solved

**Comment by givanz** [comment]
Thanks for the suggestion, in the last update plus button is moved outside if the element height is smaller than 50px.

**Comment by oleteacher** [comment]
"Thanks for the suggestion, in the last update plus button is moved outside if the element height is smaller than 50px."

That is good. But, many fontawesome icons used in footers, etc are way less than 50px. The + icons makes it virtually impossible to use fontawesome:)

**Comment by givanz** [comment]
The plus button is shown outside for all elements that are less than 50px in height, it should work for small icons also.
